crates/miden-lib/src/note/utils.rs

```diff
@@ -1,12 +1,8 @@
-use miden_objects::{
-    account::AccountId,
-    asset::Asset,
-    note::{NoteExecutionMode, NoteInputs, NoteRecipient, NoteTag, NoteType},
-    NoteError, Word,
+use crate::note::{
+    scripts, AccountId, Asset, NoteError, NoteExecutionMode, NoteInputs, NoteRecipient, NoteTag,
+    NoteType, Word,
```

Comment:
I think #1145 is just about replacing import with re-exports from the root. Here writing `use crate::note::{...}` is the same as `use super::{...}`, which I don't think is desirable.
So I think we should only replace imports with crate imports when that import is re-exported at the root, i.e. if we can do `use crate::{...}`.